tokio/src/runtime/builder.rs

```diff
@@ -109,6 +109,8 @@ pub struct Builder {
 
     #[cfg(tokio_unstable)]
     pub(super) unhandled_panic: UnhandledPanic,
+    #[cfg(tokio_unstable)]
+    pub(super) coop_budget: u8,
```

Comment:
Is there any particular reason to limit this to `u8`?

Reply:
I guess @carllerche can help answer this

Reply:
Most likely that's because the current hard-coded limit is less than 256, so there's no reason to make it any larger than u8.